UNRELEASED.md

```diff
@@ -2,6 +2,10 @@
 
 > Place your changes below this line.
 
+**Fixed:**
+- bpk-component-accordion:
+  - Improved accessibility by removing invalid `aria-level="3"` attribute on `BpkAccordionItem`.
```

Comment:
This property wasn't actually achieving anything as it was invalid, so I don't think we need to even have an entry 🤷

Reply:
I was just thinking just in case anybody is using deep rendering in their snapshots, causing them to break, this will explain why

Reply:
I guess if a consumer is skipping a test or supressing any accessibility tests for using the component this will allow them to re-enable or remove any skipping